components/FormikFields/PaperInputPicker/index.js

```diff
@@ -1,11 +1,11 @@
 import { Spinner } from 'native-base';
 import * as React from 'react';
-import {
-  Text,
-  View,
-  StyleSheet,
-  TouchableWithoutFeedback
+import { StyleSheet, TouchableWithoutFeedback, Image, Text, View
 } from 'react-native';
+<<<<<<< HEAD
+=======
+
+>>>>>>> ff2a2e07baacbb67a351accc9e33004fa1f59bf5
```

Comment:
@jamesgiegerich94 next time be careful to merge all conflicts before you commit!